underlay/src/main/resources/config/display/aouSR2023Q3R2/criteriaselector/surveyBasics/surveyBasics.json

```diff
@@ -57,5 +57,11 @@
       }
     }
   ],
-  "multiSelect": true
+  "multiSelect": false,
```

Comment:
Is there a reason you are turning off multi select?

Reply:
Yes, because the UI does not currently support multi-select and instance-level modifiers at the same time (e.g. see Labs & Measurements) because the modifier is specified in the cohort-level overview, and there's only one.
![Screenshot 2024-05-13 at 11 33 59 AM](https://github.com/DataBiosphere/tanagra/assets/12446128/9b7f001e-dcbb-4e21-81d1-8571783478f3)